platform/broadcom/sonic-platform-modules-accton/debian/sonic-platform-accton-as7926-40xfb.install

```diff
@@ -0,0 +1,3 @@
+as7926-40xfb/sonic_platform-1.0-py3-none-any.whl usr/share/sonic/device/x86_64-accton_as7926_40xfb-r0
+as7926-40xfb/sonic_platform-1.0-py2-none-any.whl usr/share/sonic/device/x86_64-accton_as7926_40xfb-r0
```

Comment:
Soon, the master branch will be moving to Debian Bullseye, and Python 2 support will be completely removed. The 202012 branch should have support for using the Python 3 version of the sonic-platform package, if it exists. Therefore, I suggest only installing the Python 3 version of the package. Otherwise, you will need to remove the installation of the Python 2 package in the near future on the master branch.